library/blob.c

```diff
@@ -1057,6 +1057,84 @@ static int eblob_blob_iterator(struct eblob_iterate_priv *iter_priv)
 	return err;
 }
 
+
+/**
+ * eblob_disk_control_cmp_by_position() - compares dc's by position in a blob file
+ */
+static int eblob_disk_control_cmp_by_position(const void *lhs, const void *rhs) {
+	const struct eblob_disk_control *left = (const struct eblob_disk_control *)lhs;
```

Comment:
Not sure, in C we can do implicit upcast from `void *` (`const struct eblob_disk_control *left = lhs;`)

Reply:
we can